src/core/gosling.schema.ts

```diff
@@ -1,7 +1,9 @@
 import { Chromosome } from './utils/chrom-size';
 
 /* ----------------------------- ROOT SPEC ----------------------------- */
-export type GoslingSpec = RootSpecWithSingleView | RootSpecWithMultipleViews;
+export type GoslingSpec =
+    | (RootSpecWithSingleView & ResponsiveSpecOfSingleView)
```

Comment:
TODO: Define specific types for this since this union is applied several times in this file.